lib/spack/spack/cmd/create.py

```diff
@@ -730,6 +749,38 @@ def get_url(args):
     return url
 
 
+def is_git_url(url):
+    """Check if the URL is likely to be a git repository. The code doesn't attempt
+    to clone the repository!
+
+    Args:
+        url (str): The url to check
+
+    Returns:
+        bool: True if it seems to be a git repository
+    """
+
+    try:
+        spack.util.url.parse_git_url(url)
+    except ValueError:
+        return False
+    else:
+        return True
+
+
+def force_git_url(url):
+    """Dummy replacement for `is_git_url` to always return True.
+
+    Args:
+        url (str): The url to check
+
+    Returns:
+        bool: always True
+    """
+
+    return True
```

Comment:
This is irrelevant given the current (May 18th) implementation of `is_git_url()`, which will return `True` even for an archive file.